mkdocs.yml

```diff
@@ -0,0 +1,66 @@
+## Project information
+site_name: Python SDK
```

Comment:
Kili Python SDK

Reply:
for that one, I removed the 'Kili' in order not to see 'Kili Kili python SDK' as we have the logo just next to it...
<img width="378" alt="Capture d’écran 2022-04-27 à 10 19 05" src="https://user-images.githubusercontent.com/82399630/165473966-c7a2dbcf-58fd-4b4f-8f0c-fa669ca26708.png">

Reply:
indeed.